chatcommands.lua

```diff
@@ -391,26 +398,35 @@ minetest.register_chatcommand("area_info", {
 				area_num = area_num + 1
 			end
 		end
-		local count_line = ("You have %d area%s"):format(
-			area_num, area_num == 1 and "" or "s")
-		if privs.areas then
-			count_line = count_line..
-				" and have no area protection limits."
-		elseif can_prot then
-			count_line = count_line..(", out of a maximum of %d.")
-				:format(max_count)
-		end
+		local count_line = privs.areas and 
+			((area_num <= 1) and
+			S("You have @1 area and have no area "..
```

Comment:
Also combining strings here would make the translator's and programmer's life much easier:

- `S("You have @1 areas.")`
- `S("Limit: no area count limit")`
- `S("Limit: @1 areas")`